.travis.yml

```diff
@@ -1,11 +1,24 @@
+sudo: required
 language: python
 python: 3.6
-env:
-  - PIPENV_IGNORE_VIRTUALENVS=1
+services:
+  - docker
+branches:
+  only:
+    - master
+    - job-dsl
 install:
   - pip install -r pipenv.txt
-script:
   - pipenv install --dev
-  - pipenv run flake8
+  - pipenv check
+jobs:
+  include:
+    - stage: Linting
+      script:
+        - pipenv install flake8
```

Comment:
We shouldn't have to resort to this, if we do the global/shared-in-jobs install correctly.

Reply:
Both of your jobs are overriding the global `script`. You could move `pipenv install --dev` to the `install` section, and move `pipenv check` to the linting script.

Reply:
Fixed, I think.

Reply:
Fixed, I think.